### DISCUSSION TITLE
Death from disconnecting counts as a normal death

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14298] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The player character dying due to diconnecting counts as a regular death and causes the character to lose skills, along with their inventory (the inventory gets preserved in a duffel bag but equipped genes do not get preserved). It's expected for the player to not suffer any consequences from dying after disconnecting (unless they actually die due to enemies, pressure etc.).

### Reproduction steps

1. Start a multiplayer campaign with very short kill disconnect timer
2. Have 2nd player with gained skills and inventory items disconnect
3. Wait until player character dies from brain death
4. Begin a new round and have the 2nd player reconnect

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
